app/templates/me/index.hbs

```diff
@@ -30,9 +30,9 @@
     </div>
 
     <p>
-        If you want to use package commands from the command line, you'll need a
-        <code>~/.cargo/credentials</code>, the first step to creating this
-        is to generate a new token using the button above.
+        If you want to use package commands from the command line, you will need to
+        login with <code>cargo login</code> using one of the tokens listed below. <br/>
+        When working in shared environments, supplying the token on the command line could expose it to other users, so it is more secure to supply a token when prompted. 
```

Comment:
I'll do that. I was just trying to use the wording suggested by the original issue.